### PR TITLE
BITMAKER-853: Fix logger and describe-project output

### PR DESCRIPTION
**Pull request by rodp63**
#### Ticket url(s):
- https://tasks.bitmaker.dev/issues/853

#### Description:
- Fix the _bitmaker-entrypoint_ logger.
- The `bm-describe-project` command must output to stdout. 

#### Checklist:
- [X] Check that only one commit is added in the PR with the correct format
- [X] Rebase my branch
- [X] Run `pytest`

### CODE REVIEW

File: bm_scrapy/commands/describe_project.py [hunk 1]
```diff
@@ -47,4 +46,4 @@ def run(self, args, opts):
                 universal_newlines=True,
             )
             result["image"] = output
-        logger(json.dumps(result))
+        print(json.dumps(result))
```
Comment (renato262) [inline bm_scrapy/commands/describe_project.py:49]: @rodp63 why `print`?

File: bm_scrapy/producer.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import json
 import logging
 
-from bm_scrapy import logger
+
```
Comment (renato262) [inline bm_scrapy/producer.py]: Remove this empty line please

File: bm_scrapy/producer.py [hunk 3]
```diff
@@ -18,15 +18,15 @@ def connect_kafka_producer():
         _producer = KafkaProducer(
             bootstrap_servers=bootstrap_servers,
             value_serializer=lambda x: json.dumps(x).encode("utf-8"),
+            api_version=(0, 10),
```
Comment (renato262) [inline bm_scrapy/producer.py:20]: Why are we adding api version?

File: bm_scrapy/commands/describe_project.py (hunk 1 above)
Comment (rodp63) [reply bm_scrapy/commands/describe_project.py:49]: In the cli, we use docker to get the spider list, the returned value is the stdout of the container, no the logs.

File: bm_scrapy/producer.py (hunk 3 above)
Comment (rodp63) [reply bm_scrapy/producer.py:20]: It turns out that we actually need this field 😅

File: bm_scrapy/producer.py (hunk 2 above)
Comment (rodp63) [reply bm_scrapy/producer.py]: There are 2 lines after the imports 🤔

Comment (renato262) [reply bm_scrapy/producer.py]: but kafka is imported after this?

Comment (rodp63) [reply bm_scrapy/producer.py]: You're rigth. Done

**Review by renato262 — APPROVED** [review]

**Pull request merged**